dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/citation/CitationData.java

```diff
@@ -33,7 +38,7 @@
     private List<String> kindsOfData = new ArrayList<>();
     private List<String> languages = new ArrayList<>();
     private List<String> spatialCoverages = new ArrayList<>();
-
+    private List<String> otherIds = new ArrayList<>();
     private List<DatasetField> optionalValues = new ArrayList<>();
     private int optionalURLcount = 0;
```

Comment:
If we dropped support for these `optionalValues` than I don't see a point in keeping this fields here